src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -3048,6 +3054,15 @@ void commit() throws IOException {
             throw new IllegalStateException("storage conflict with existing key for " + key);
           }
         } else if (writeWinner.get()) {
+          RequestMetadata requestMetadata = TracingMetadataUtils.fromCurrentContext();
+          log.log(
+              Level.INFO,
+              format(
+                  "Successfully wrote: %s; metadata: actionId - %s, actionMnemonic - %s, target - %s",
+                  key,
+                  requestMetadata.getActionId(),
+                  requestMetadata.getActionMnemonic(),
+                  requestMetadata.getTargetId()));
```

Comment:
Why is this a) assumption of grpc context and b) unrelated-to-access metrics log entry in this change?

Reply:
There is no easy way to reverse map an digest to ActionMenomnic and target pair. I needed this to find targets with indeterminism. Definitely unrelated to cas access metrics, removed will create another PR for this change.